src/main/java/cceclipseplugin/editor/DocumentManager.java

```diff
@@ -117,8 +117,11 @@ public ITextEditor getEditor(String absolutePath) {
 	 *
 	 * @return The queue of diffs that was applied.
```

Comment:
Not really. I've put it into a hashmap, and used a linkedlist instead, but in all reality, it's a queue. I just needed access to all the methods of the linkedlist.